content/en/quality_gates/explorer/saved_views.md

```diff
@@ -0,0 +1,65 @@
+---
+title: Saved Views
+kind: documentation
+further_reading:
+- link: "/quality_gates/explorer/search_syntax/"
+  tag: "Documentation"
+  text: "Learn how to create a search query"
+---
+
+{{< site-region region="gov" >}}
+<div class="alert alert-warning">Quality Gates is not available in the selected site ({{< region-param key="dd_site_name" >}}) at this time.</div>
+{{< /site-region >}}
+
+{{< callout url="#" btn_hidden="true" >}}
+Quality Gates is in public beta.
+{{< /callout >}}
+
+## Overview
+
+Saved views allow you to save the state of the [Quality Gates Explorer][2] on the [**Quality Gate Executions** page][1] and enable effective troubleshooting by providing you with access to scoped queries, relevant facets, visualization options, and the time range. 
+
+Saved views can keep track of your:
+
+- Pipeline executions and test runs
```

Comment:
update with quality gate executions and rule executions

Reply:
```suggestion
- Quality Gate executions and rule executions
```